app/src/main/res/layout/fragment_map.xml

```diff
@@ -1,38 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:gravity="center"
-    android:orientation="vertical"
     tools:context=".presentation.map.MapFragment"
-    tools:ignore="HardcodedText">
+    tools:ignore="HardcodedText,ContentDescription">
 
-    <TextView
-        android:id="@+id/mapFragmentTV"
+    <androidx.fragment.app.FragmentContainerView
+        android:id="@+id/mapFragment"
+        android:name="com.google.android.gms.maps.SupportMapFragment"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="center" />
+
+    <ImageView
+        android:id="@+id/locationIV"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Map Fragment"
-        android:textSize="@dimen/text_view_text_size"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        android:layout_gravity="top|end"
+        android:layout_marginTop="@dimen/map_icons_margin_top"
```

Comment:
Why do you need this maginTop?
Also, you should try and export these icons again. For some reason they have a pretty big empty space around them that makes the ImageView bigger than it should be. Let me know if you need some help with this.